### PR TITLE
Merge create-drop into new-direction

### PR DESCRIPTION
**Pull request by csongorszasz**
Implemented:
- CREATE DATABASE
- DROP DATABASE
- CREATE TABLE
- DROP TABLE

### CODE REVIEW

File: server_side/database_objects/dbo.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 
 class Dbo(ABC):
-    __name: str = ""
+    __constraint_name: str = ""
```
Comment (p3t9a) [inline server_side/database_objects/dbo.py]: Why was this changed?
This being an abstract class all implementations should change if you change it, but I don't see the logic in changing this fields name.

File: server_side/dbmanager.py [hunk 2]
```diff
@@ -54,9 +54,42 @@ def update_databases(self):
     #             # TO-DO: create table
```
Comment (p3t9a) [inline server_side/dbmanager.py]: Unnecessary comment

**Review by p3t9a — COMMENTED** [review]
Great work. The test cases are a nice touch. I left some comments please check them out.

File: server_side/database_objects/dbo.py (hunk 1 above)
Comment (csongorszasz) [reply server_side/database_objects/dbo.py]: It must have changed by accident while refactoring some other variable. We can leave it as __name.

File: server_side/dbmanager.py (hunk 2 above)
Comment (csongorszasz) [reply server_side/dbmanager.py]: It wasn't mine hah

**Comment by csongorszasz** [comment]
@p3t9a Ready to merge? 
Also we could merge new-direction into main after this, couldn't we?

File: server_side/dbmanager.py (hunk 2 above)
Comment (p3t9a) [reply server_side/dbmanager.py]: my bad sorry

**Comment by p3t9a** [comment]
LGTM let's merge it into main too

**Review by p3t9a — APPROVED** [review]
